src/fitnesse/responders/DisabledResponder.java

```diff
@@ -17,7 +17,8 @@ protected String contentFrom(FitNesseContext context, Request request, WikiPage
     return prepareResponseDocument(context).html(request);
```

Comment:
This method can be removed. It does that same thing as the method it overrides

Reply:
> This method can be removed. It does that same thing as the method it overrides

No `DisabledResponder` sets a different template which differs from `BasicResponder`.

Reply:
I meant the method `contentFrom()` it, which you have not touched yet. Sorry I wasn't more clear. The implementation of `prepareResponseDocument()` is indeed different, but `contentFrom` not, I believe.